Assets/Scripts/Rendering/State.cs

```diff
@@ -19,10 +19,35 @@ public void AddNewMqClient(MQ.Client newMqClient)
         qmgrs.Add(newMqClient, null);
     }
 
+    /* 
+    * Get Number of QM Registered -> For Navigation Check box
+    */
+    public int GetNumberMQ()
```

Comment:
I would name this method `GetNumberOfQueueManagers` (watch out for the difference in MQ and QM. MQ = message queue, that's the whole system. But QM = Queue Manager / qmgr, which is what we are dealing with now. Right now we are dealing with 1 MQ installation on server = 1 QM, but on local instances there can be situations where 1 MQ installation = N QMs)